modules/validations/invalid_format.js

```diff
@@ -51,7 +51,7 @@ export function validationFormatting() {
                     if (edtfFromOSM) {
                         let label;
                         try {
-                            label = edtf.default(edtfFromOSM).format(localizer.languageCode());
+                            label = edtf.default(edtfFromOSM).format(localizer.localeCode());
```

Comment:
Upon code inspection, edtf.js seems to expect a single locale string here, not an array of them like the built-in functions.